azkaban-web-server/src/test/java/azkaban/webapp/AzkabanWebServerTest.java

```diff
@@ -147,8 +148,7 @@ public void testInjection() throws Exception {
     assertSingleton(ExecutorEventsDao.class, injector);
     assertSingleton(ActiveExecutingFlowsDao.class, injector);
     assertSingleton(FetchActiveFlowDao.class, injector);
-    assertSingleton(AzkabanWebServer.class, injector);
-
+    assertSingleton(H2FileDataSource.class, injector);
```

Comment:
How about the mysql datasource?

Reply:
Mysql dataSource cannot be added here, since Guice only have h2 Props, which has conflicts against Mysql Props.

Reply:
Can it be tested separately? It can be in a separate PR.